# params/lidar_move_example.yaml
# Common parameters for limo_example
---

# Dynamic parameters cannot have a namespace
min_angle_deg: -0.5
max_angle_deg: 0.5
linear_x:  0.2
min_dist:  0.3